src/python/vehicle.py

```diff
@@ -1,51 +1,79 @@
 from typing import Dict, Callable
+from command_line import call_vehicle_to_generate_loss_json
 import utils
 import tensorflow as tf
 import numpy as np
-import random
-import json
 
 
-def generate_loss_function(path_to_spec:str, function_name:str, resources:Dict[str,any]) -> Callable:
+def generate_loss_function(specification:str,
+                            function_name:str,
+                            resources:Dict[str,any],
+                            quantifier_sampling:Dict[str,Callable]=None) -> Callable:
     '''
     pathToSpec: path to the vehicle spec .vcl file
     functionName: name of the function for which we want to create the loss function
     resources: dictionary mapping from the name of the resources declared in the spec to the python implementation
```

Comment:
Missing documentation for new `quantifier_sampling` parameter?

Reply:
Updated